pkg/appfile/storage/driver/local.go

```diff
@@ -0,0 +1,131 @@
+package driver
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/oam-dev/kubevela/pkg/appfile"
+	"github.com/oam-dev/kubevela/pkg/appfile/template"
+	"github.com/oam-dev/kubevela/pkg/utils/env"
+	"github.com/oam-dev/kubevela/pkg/utils/system"
+)
+
+// LocalDriverName is local storage driver name
+const LocalDriverName = "Local"
+
+// Local Storage
+type Local struct {
+}
```

Comment:
Are you going to add fields in struct `Local`, or just by-design?

Reply:
Each storage type will have a struct and implement `type Driver interface`.
The local storage type has no fields to add. To implement the general driver interface.
We just need to use the interface of `type Storage struct`.